documentation/persistent-vols-claims.md

```diff
@@ -2,9 +2,9 @@
 title: Persistent Volumes & Persistent Volumes Claims
 ---
 
-In case of Kubernetes Volumes we know that once the Pod is deleted the specification of the volume in the Pod is also lost. Even though VMDK file persists but from Kubernetes perspective the volume is deleted.
+In case of Kubernetes Volumes, we know that once the Pod is deleted the specification of the volume in the Pod is also lost. Even though VMDK file persists but from Kubernetes perspective the volume is deleted.
  
-Persistent Volumes API resource solves this problem where PVs have lifecycle independent of the Pods and not created when Pod is run. PVs are unit of storage which we provision in advance, they are Kubernetes objects backed by some storage, vSphere in this case. PVs are created, deleted using kubectl commands.
+Persistent Volumes API resource solves this problem where PVs have lifecycle independent of the Pods and not created when Pod is spawned. PVs are units of storage provisioned in advance, they are Kubernetes objects backed by vSphere storage. PVs are created, deleted using kubectl commands.
```

Comment:
`vSphere storage` -> we need to the standard term here, vSphere storage for kubernetes. But Kubernetes word is redundant. Not sure if we can use any abbreviation here for readability.

Reply:
+1

Reply:
Here `vSphere storage` mean,  actual vsphere datastores (vSan, VVol, vmfs, nfs), and not the `vSphere Storage for Kubernetes` Products.